components/GradientButton.tsx

```diff
@@ -1,38 +1,49 @@
 import { PropsWithChildren } from "react";
 import { twMerge } from "tailwind-merge";
+import { ReactNodeLike } from "prop-types";
 
 interface GradientButtonProps {
   onClick?: () => void;
+  href?: string;
   className?: string;
   elementId?: string;
   outerDivClassName?: string;
+  small?: boolean;
 }
 
 const GradientButton = ({
   onClick,
+  href,
   className,
   elementId,
   outerDivClassName,
   children,
+  small,
 }: PropsWithChildren<GradientButtonProps>) => {
-  return (
-    <div
+  const wrapClassName = twMerge(
+    "relative bg-gradient2 rounded-[100px] p-[1px] w-fit h-fit hover:opacity-90",
+    outerDivClassName,
+  );
+  const wrap = (children: ReactNodeLike) =>
+    href ? (
+      <a href={href} className={wrapClassName}>
+        {children}
+      </a>
+    ) : (
+      <div className={wrapClassName}>{children}</div>
+    );
+  return wrap(
+    <button
```

Comment:
not sure about nesting a button in a `a` tag, aren't you getting an html error?

Reply:
I saw it done elsewhere in the app so I did it here too. 😅